sympy/assumptions/tests/test_query.py

```diff
@@ -2026,21 +2024,6 @@ def test_key_extensibility():
     # make sure the key is not defined
     raises(AttributeError, lambda: ask(Q.my_key(x)))
 
-    # Old handler system
-    class MyAskHandler(AskHandler):
-        @staticmethod
```

Comment:
We should keep the tests for the old handler system while it is deprecated. You can use `warns_deprecated_sympy` to test that it still works and also gives a warning.